metadata.go

```diff
@@ -331,6 +331,9 @@ func getKeyspaceMetadata(
 		`,
 		keyspaceName,
 	)
+	// Set a routing key to avoid GetRoutingKey from computing the routing key
+	// TODO use a separate connection (pool) for system keyspace queries.
+	query.RoutingKey([]byte{})
```

Comment:
I see. I have ideas to replace the pluggable connection pool with pluggable host selectors for a query, so that should be replaceable on a per query basis.